modules/developer_manual/pages/app/fundamentals/publishing.adoc

```diff
@@ -164,7 +164,8 @@ Apps with the `Enterprise` tag are official ownCloud enterprise apps.
 These can only be uploaded by ownCloud itself and represent ownCloud
 Enterprise Edition features.
 
-image:app/app-tile-enterprise.jpg[ownCloud "Enterprise" tag]
+.ownCloud "Enterprise" tag
+image:app/app-tile-enterprise.jpg[The ownCloud 'Enterprise' Tag]
 
 [[verified]]
```

Comment:
why don´t you have, for the sake of completeness:
``verified`` --> ``publishing_verified`` ?

Reply:
See above.